app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+from typing import List, Dict
+
+
+def go_to_cafe(friends: List[Dict], cafe: Cafe) -> str:
+    masks_to_buy: int = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The error messages for NotVaccinatedError and OutdatedVaccineError are the same. Consider using descriptive error messages as per the checklist: 'Use descriptive error messages'.